sootup.java.bytecode/src/main/java/sootup/java/bytecode/interceptors/EmptySwitchEliminator.java

```diff
@@ -42,15 +42,13 @@ public class EmptySwitchEliminator implements BodyInterceptor {
   @Override
   public void interceptBody(@Nonnull Body.BodyBuilder builder, @Nonnull View<?> view) {
     // Iterate all stmts in the body
-
-    for (Stmt stmt : builder.getStmtGraph()) {
+    for (Stmt stmt : new ArrayList<>(builder.getStmtGraph().getNodes())) {
```

Comment:
why?

Reply:
a) order is not relevant (faster node access) b) changing a collection while iterating over it -> Concurrenmodificationexception